di_website/visualisation/models.py

```diff
@@ -0,0 +1,51 @@
+from django.db import models
+from django.shortcuts import redirect
+
+from wagtail.core.models import Page
+from wagtail.core.fields import StreamField
+from wagtail.admin.edit_handlers import StreamFieldPanel
+
+from .blocks import ChartBlock
+
+
+class VisualisationsPage(Page):
+    """
+    Parent page for all visualisations
+    """
+    parent_page_types = ['home.HomePage']
+    subpage_types = ['visualisation.ChartPage']
+
+    class Meta:
+        verbose_name = 'Visualisations Page'
+
+    def get_sitemap_urls(self, request):
+        return []
+
+    def serve(self, request, *args, **kwargs):
```

Comment:
@edwinmp 

As this is in essence a folder for the chart pages, I would say that the return value for the listing page would legitimately be a 404, as it should never be viewable:

```
from django.http import Http404

def serve(request):
    raise Http404()
```